doc/reference/interactive_console.rst

```diff
@@ -120,3 +125,22 @@ call was ``console.set_default_output('lua')``.
                 | :code:`...`
 
             -   :code:`\x81 \xa3 \x6b \x65 \x79 \x01`
+
+.. _interactive_console-shortcuts:
+
+Keyboard shortcuts
+------------------
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 1
+
+        *   - Keyboard shortcut
+            - Effect
+
+        *  - ``CTL+C``
```

Comment:
It usually abbreviated as Ctrl, not Ctl AFAIK.

AFAIR, we should use some template for keydoard keys.

Reply:
Fixed typo. Thanks for noticing!